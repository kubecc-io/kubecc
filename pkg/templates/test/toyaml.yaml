strTest: {{ .Spec.StrTest | toYaml }}
numTest: {{ .Spec.NumTest | toYaml }}
stringSliceTest:
{{ .Spec.StringSliceTest | toYaml | indent 2 }}
nestedStruct:
{{ .Spec.NestedStruct | toYaml | indent 2 }}