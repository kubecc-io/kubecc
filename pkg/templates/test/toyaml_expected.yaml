strTest: test
numTest: 5
stringSliceTest:
- a
- b
- c
nestedStruct:
  numTest: 6
  nestedStruct:
    strTest: test2
    stringSliceTest:
    - |-
      line 1
      line 2
      line 3
    - test
